src/pkg/utils/misc.go

```diff
@@ -34,3 +34,13 @@ func Retry(fn func() error, retries int, delay time.Duration) (err error) {
 
 	return err
 }
+
+// Insert returns a new slice with the element inserted at the given index
+func Insert[T any](slice []T, index int, element T) []T {
+	if len(slice) == index { // nil or empty slice or after last element
```

Comment:
Though it's not universal, we try to follow line comments above lines, https://github.com/uber-go/guide/blob/master/style.md.

Reply:
Fixed